### ISSUE TITLE
Is this still maintained?

### ISSUE THREAD

**Issue by belhassen07** [hibiken/react-places-autocomplete#296]
I'm having multiple issues here: 
- Restrcting bound as the #143 issue decribes, there's no solution implemented for this. 
- The last commit is about 2 years ago which is a concern
- Warning: componentWillReceiveProps has been renamed, and is not recommended for use. That means the library won't work with the new react version.

**Comment by zhil** [comment]
@hibiken ping

**Comment by hibiken** [comment]
@belhassen07 and @zhil , thank you for opening this issue.

I've been struggling to find time and motivation to work on this project since I no longer use React in my day-to-day work.

I have a few thoughts:
* If there's a fork that's well maintained, I'd happy to list that in the README of this repo.
* If anyone is willing to be added as a maintainer of this project, please email me directly (ken.hibino7@gmail.com). I may ask your thoughts on the roadmap of the project.

**Comment by zhil** [comment]
few candidates for maintained fork / maintainers (according to network graph)

@travelboss @afreix @wadehrarshpreet @innovativemetrics @chamchisand @hkdahal @alvinhuynh @InTouchGPS @outdoorjournal

Personally I dont have enough reactJS skills to maintain lib, would be nice if some fork owner will do it though.

**Comment by sambokai** [comment]
@hibiken This should be more visible. I've added it to the README in this PR. https://github.com/hibiken/react-places-autocomplete/pull/316

**Comment by hibiken** [comment]
@sambokai thanks for all your contributions! Would you mind if I add you as a collaborator to this project?

**Comment by sambokai** [comment]
Sure, why not. At least for occasional reviews and merging, but I will not be able to commit a substantial amount of time. Would be nice to find a few more maintainers. Messaging the people in @zhil's list would be start. I'll do that.

**Comment by hibiken** [comment]
I agree. Thank you for the contributions you've made so far! Please don't feel pressured to review or dedicate time on the project :)

**Comment by hibiken** [comment]
I've created a project board to keep track of what's needed to be done, so if anyone wants to contribute, please take a look at the current project.

**Comment by StephaneP** [comment]
interested in helping out

**Comment by Paul-Taiwo** [comment]
I am interested in helping out too.

**Comment by hibiken** [comment]
Thanks for your interest @StephaneP and @Paul-Taiwo 😄 
Is there any issues you'd like to take? Good start may be  #326, #318, or any open issues that has a number of +1s.

**Comment by Paul-Taiwo** [comment]
Thank you, Ken, for giving me this opportunity. 😄
I would like to know if there is a convention for creating PR's and commit
messages.

**Comment by Ahmdrza** [comment]
@hibiken Hi, are you still looking for maintainers?

**Comment by mding5692** [comment]
Interested in helping out too, one of my company's projects uses this as a dependency so willing to contribute as much as needed

**Comment by khanzzirfan** [comment]
Interested in helping out too!. Please add me to the list as well.

**Comment by patrick934** [comment]
Hi @hibiken
I'm interested in helping out as well

**Comment by dougschallmoser** [comment]
I'm interested in helping out where I can!

**Comment by missinglink** [comment]
The maintainers of [Pelias](https://github.com/pelias/pelias) would be happy to provide some ongoing financial support/technical expertise on the proviso that the library also supported open-source and open-data geocoders.

**Comment by imtmh** [comment]
Hey, I am too interested in helping out. 
Great work btw.

**Comment by Ebugo** [comment]
Nice work here!
I'm interested in helping with maintenance too, it's still open right?

**Comment by morfeli** [comment]
If there is anything I may do to assist please let me know.

**Comment by HakanSungur** [comment]
Its return bad request. please update your codes.

**Comment by twoelevenjay** [comment]
@hibiken I am interested in being a maintainer. I have now been working with ReactJS daily for a year and I am ready to step it up.

**Comment by Dhruvinhp** [comment]
hey @hibiken i'm interested in being a maintainer, let me know if you need help

**Comment by symonbaikov** [comment]
I am interested in being a maintainer. Please let me know.

**Comment by davidcoleman007** [comment]
@MostOfLuck please contact the owner.

https://github.com/hibiken/react-places-autocomplete?tab=readme-ov-file#we-are-looking-for-maintainers

![image](https://github.com/user-attachments/assets/26a124d7-0510-48cf-9a46-1741a7e11192)

**Comment by shivakumarbdoddamani** [comment]
@hibiken who is maintaining this? If you are not then check the mail sent by me.

**Comment by shivakumarbdoddamani** [comment]
@belhassen07 i don't think it is maintained anymore.

**Comment by MatteoGiacomo** [comment]
I just released a new package with the new Google Maps API. 
https://www.npmjs.com/package/react-places-suggestion?activeTab=readme
